libraries/lib-registries/Registry.h

```diff
@@ -122,21 +124,34 @@ namespace Registry {
       ~SingleItem() override = 0;
    };
 
-   // Common abstract base class for items that group other items
+   //! Common abstract base class for items that group other items
    struct REGISTRIES_API GroupItem : BaseItem {
       using BaseItem::BaseItem;
 
-      // Construction from an internal name and a previously built-up
-      // vector of pointers
+      //! Construction from an internal name and a previously built-up
+      //! vector of pointers
       GroupItem( const Identifier &internalName, BaseItemPtrs &&items_ )
          : BaseItem{ internalName }, items{ std::move( items_ ) }
       {}
       GroupItem( const GroupItem& ) PROHIBITED;
       ~GroupItem() override = 0;
 
-      // Whether the item is non-significant for path naming
-      // when it also has an empty name
-      virtual bool Transparent() const = 0;
+      //! Choose treatment of the children of the group when merging trees
+      enum Ordering {
```

Comment:
Have you considered using `enum class` ? One wouldn't get false positives searching the code for 
e.g. `Ordering::Strong` as opposed to just `Strong`, for example.